src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurStandardAccountingExtractValidationServiceImpl.java

```diff
@@ -0,0 +1,90 @@
+package edu.cornell.kfs.concur.batch.service.impl;
+
+import java.sql.Date;
+
+import org.apache.commons.lang.StringUtils;
+import org.apache.log4j.Logger;
+import org.kuali.rice.core.api.util.type.KualiDecimal;
+
+import edu.cornell.kfs.concur.ConcurConstants;
+import edu.cornell.kfs.concur.batch.businessobject.ConcurStandardAccountingExtractDetailLine;
+import edu.cornell.kfs.concur.batch.businessobject.ConcurStandardAccountingExtractFile;
+import edu.cornell.kfs.concur.batch.service.ConcurStandardAccountingExtractValidationService;
+
+public class ConcurStandardAccountingExtractValidationServiceImpl implements ConcurStandardAccountingExtractValidationService {
+    private static final Logger LOG = Logger.getLogger(ConcurStandardAccountingExtractValidationServiceImpl.class);
+    
+    @Override
+    public boolean validateConcurStandardAccountExtractFile(ConcurStandardAccountingExtractFile concurStandardAccountingExtractFile) {
+        boolean valid = validateDetailCount(concurStandardAccountingExtractFile);
+        valid = validateAmounts(concurStandardAccountingExtractFile) && valid;
```

Comment:
I was to run validateAmounts even if validi is already false, so we can log the other errors.